thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -287,6 +286,34 @@ public Response generateAnomaliesInRange(@PathParam("id") String id,
     return Response.ok().build();
   }
 
+  @POST
+  @Path("/{id}/offline_analysis")
+  public Response getOutliersInTrainingData(@PathParam("id") @NotNull long id,
+      @QueryParam("time") String analysisTimeIso) throws Exception {
```

Comment:
Rename to monitoringTimeIso? Because we always name the window for analysis as monitoringWindowSomething; using the same naming convention could prevent some misunderstanding.